modules/storage-azure-create-storage-class.adoc

```diff
@@ -33,6 +33,11 @@ storage account SKU tier. Valid options are `Premium_LRS`, `Standard_LRS`,
 
 ... Enter the kind of account. Valid options are `shared`, `dedicated,`
 and `managed`.
++
+[IMPORTANT]
+====
+Red Hat does not support the use of `kind: Dedicated` in the storage class. {product-title} creates a managed disk for machine OS, or root, disks. However, Azure Disk does not allow the use of both managed and unmanaged disks on a node. Therefore, unmanaged disks cannot be attached to {product-title} nodes.
+====
```

Comment:
Based on [this comment](https://bugzilla.redhat.com/show_bug.cgi?id=1918270#c7) in the BZ.

Reply:
seems the volumes created with kind: Shared are the unmanaged disks too.

Reply:
Same question here for @jsafrane: Do we need to update this note to something like the following?

> Red Hat only supports the use of `kind: Managed` in the storage class...etc.

Reply:
Same question here for @sferich888 - looking for comments or ack to add a note to indicate that RH only supports `kind: Managed` and not `Shared` or `Dedicated`

Reply:
Yes, it should say that Shared is not supported too.